docs/user_guides/fs/storage_connector/creation/redshift.md

```diff
@@ -44,17 +43,20 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
 5. The database name.
 6. The database port.
 7. The database username, here you have the possibility to let Hopsworks auto-create the username for you.
-8. Optionally provide the database group and table to point the connector to.
+8. Database Driver (optional): You can use the default JDBC Redshift Driver `com.amazon.redshift.jdbc42.Driver` 
+   included in Hopsworks or set a different driver (More on this later).
+9. Optionally provide the database group and table to point the connector to.
 9. Set the appropriate authentication method.
 
 <figure markdown>
   ![Redshift Connector Creation](../../../../assets/images/guides/fs/storage_connector/redshift_creation.png)
   <figcaption>Redshift Connector Creation Form</figcaption>
 </figure>
 
-### Step 3: Upload the Redshift database driver
+### Step 3: Upload the Redshift database driver (optional)
 
-With regards to the database driver, the library to interact with Redshift is not included in Hopsworks - you need to upload the driver yourself. First, you need to [download the library](https://docs.aws.amazon.com/redshift/latest/mgmt/welcome.html). Select the driver version without the AWS SDK.
+By default, JDBC Redshift driver library `redshift-jdbc42`, is included in Hopsworks to interact with Redshift. If you wish to use the same you can skip this step. Otherwise, to use a 
```

Comment:
I would write - The `redshift-jdbc42` JDBC driver is included by default in the Hopsworks distribution. If you wish to use a different driver, you need to upload it on Hopsworks and add it as a dependency of Jobs and Jupyter Notebooks that need it.

Reply:
fixed